src/main/java/org/mule/service/scheduler/internal/DefaultSchedulerService.java

```diff
@@ -54,7 +56,11 @@
  */
 public class DefaultSchedulerService implements SchedulerService, Startable, Stoppable {
 
+  private static final String USAGE_TRACE_INTERVAL_SECS_PROPERTY = "mule.scheduler.usageTraceIntervalSecs";
```

Comment:
Does it not make sense to have a simple `mule.scheduler.enableUsageLog` that uses a default of 1 second? (and optional a separate property for the frequency. Rather than than requiring `mule.scheduler.usageTraceIntervalSecs=1`?  (I guess thats then 2 properties though..)